modules/aleph/includes/aleph.reservation.inc

```diff
@@ -260,17 +248,16 @@ function aleph_reservation_create($account, $id) {
   $interest_period = aleph_reservation_default_interest_period();
 
   $end_request_date = new DateTime();
-  $end_request_date->add(new DateInterval('P' . $interest_period . 'D'));
+  $end_request_date->add(new DateInterval('P' . $options['interest_period'] . 'D'));
```

Comment:
Are you 100% that this key is always available? (if this basically falls back on the return-value from aleph_reservation_default_options() i guess we're ok - but in that case it might actually be null ... or?)

Reply:
There's no "none" option or similar, so it will always be valid.